README.md

```diff
@@ -1 +1,82 @@
-# stock-data
\ No newline at end of file
+# Stock Data
+
+> This is a web app built with the financial api data that fetches the different companies listed at the major stock exchanges in USA, like NASDAQ and NYSE. The app fetches data from two different APIs , combines them and then outputs the data to the screen.
+
+##  Video Documentation
+
+[Video](./src/images/video1616425580.mp4)
```

Comment:
- [ ] I recommend adding only this link: https://drive.google.com/file/d/1-wqEW2AIMUOy1W0_-kTjYyX1YCuAXVjw/view?usp=sharing as a link for video explanation instead of this link: ./src/images/video1616425580.mp4, which a user has to download. Because it is not always comfortable to download for users.